ocpp/async_message_handler.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+from abc import abstractmethod
+
+
+class AsyncMessageHandler(ABC):
```

Comment:
Everyone loves their interfaces now! 😄

Reply:
Exposure to Go is paying off!

Reply:
Go does not get the credit there I'm afraid ;)